src/DynamoCore/Interfaces/IPackage.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Interfaces
+{
+    /// <summary>
+    /// Interface that contains basic properties of a Dynamo package
+    /// </summary>
+    public interface IPackage
+    {
+        /// <summary>
+        /// Name of the package
+        /// </summary>
+        string Name { get; set; }
+
+        /// <summary>
+        /// Version of the package
+        /// </summary>
+        string VersionName { get; set; }
+
+        /// <summary>
+        /// Assemblies contained in this package that are node libraries
```

Comment:
lets get more specific about this - I'm not sure if this means just zero touch assemblies, or assemblies containing NodeModels or those containing NodeModel customizations? Can we find out?